Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Estimate.hs

```diff
@@ -143,5 +143,19 @@ mkAdditionalBreakups mkPrice mkBreakupItem distance farePolicy = do
       let (perExtraKmFareSection :| _) = NE.sortBy (comparing (.startDistance)) det.perExtraKmRateSections
           perExtraKmFareCaption = "EXTRA_PER_KM_FARE"
           perExtraKmFareItem = mkBreakupItem perExtraKmFareCaption (mkPrice $ roundToIntegral perExtraKmFareSection.perExtraKmRate)
-      [perExtraKmFareItem]
-    mkAdditionalSlabBreakups _ = []
+
+      let waitingOrPickupChargesCaption = "WAITING_OR_PICKUP_CHARGES"
+          mbWatingChargeValue =
+            (det.waitingChargeInfo <&> (.waitingCharge)) <&> \case
+              PerMinuteWaitingCharge hpm -> roundToIntegral hpm
```

Comment:
In case of `PerMinuteWaitingCharge` also, we will be returning per minute value, not the total charge?

Reply:
> In case of `PerMinuteWaitingCharge` also, we will be returning per minute value, not the total charge?

Yes, it is the issue, that before it was returning rate, not total charge